src/test/java/uk/gov/di/ipv/cri/common/library/util/SignedJWTFactoryTest.java

```diff
@@ -84,7 +84,30 @@ void shouldCreateASignedJwtSuccessfullyWithCorrectKeyIdStructure()
         System.out.println(new ObjectMapper().writeValueAsString(signedJWT));
         assertEquals("did", keyIDArray[0]);
         assertEquals("web", keyIDArray[1]);
-        assertEquals("issuer", keyIDArray[2]);
+        assertEquals(
+                "issuer#2078472f90d737929e5a580594dc563900af22b08161258760581e7f16b3d142",
+                keyIDArray[2]);
+    }
+
+    @Test
+    void shouldRemoveHttpsFromTheIssuerAndSuccessfullyCorrectKeyIdStructure()
+            throws JOSEException, InvalidKeySpecException, NoSuchAlgorithmException,
+                    JsonProcessingException {
+        JWTClaimsSet testClaimsSet = new JWTClaimsSet.Builder().build();
+        signedJwtFactory = new SignedJWTFactory(new ECDSASigner(getPrivateKey()));
+
+        SignedJWT signedJWT =
+                signedJwtFactory.createSignedJwt(testClaimsSet, "https://issuer", "keyId");
+        System.out.println(signedJWT.getHeader().getKeyID());
```

Comment:
Can we remove this, I suppose it was for debugging

Reply:
Yeah was just c&p'd from the above test, removed from both